pkgs/development/compilers/llvm/common/libclc.nix

```diff
@@ -16,10 +18,18 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./libclc/libclc-gnu-install-dirs.patch
-  ];
+  ] # LLVM 19 changes how host tools are looked up.
+    # Need to remove NO_DEFAULT_PATH and the PATHS arguments for find_program
+    # so CMake can actually find the tools in nativeBuildInputs.
+    # https://github.com/llvm/llvm-project/pull/105969
+    ++ lib.optional (lib.versionAtLeast release_version "19") (fetchpatch {
+      url = "https://github.com/llvm/llvm-project/pull/105969/commits/e8b910246d0c7c3d9fff994f71c6f8a48ec09a50.patch";
```

Comment:
This is not a stable URL, because if you force push to that PR, this commit might end up being garbage collected.  For unmerged PRs, we should include the patch file in Nixpkgs.

Reply:
Done.